src/User32.Desktop/User32+SPIF.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="SPIF"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        [Flags]
+        public enum SPIF
```

Comment:
Since this describes the flags that can be passed to the `SystemParametersInfo` method, the convention is to name it `SystemParametersInfoFlags`.